.github/workflows/scripts/get_remote_labels.sh

```diff
@@ -2,9 +2,9 @@
 
 source $( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/inspect_remote_img.sh
 get_remote_labels() {
-  if [[ $# -ne 2 ]]; then
-    echo '[get_remote_labels](./get_remote_labels.sh) $GH_TOKEN $IMAGE'
-    echo 'Example: get_remote_labels XXXXXXXXXXXX ghcr.io/nvidia/t5x:latest'
+  if [[ $# -ne 4 ]]; then
+    echo '[get_remote_labels](./get_remote_labels.sh) $GH_TOKEN $IMAGE $OS $ARCH'
```

Comment:
this change didn't introduce this, but could we fix this in your PR?
```suggestion
    echo 'get_remote_labels $GH_TOKEN $IMAGE $OS $ARCH'
```